### PR TITLE
update: 1.6.10-rc01 what's new

### PR DESCRIPTION
**Pull request by zamulla**

### CODE REVIEW

File: topics/whats-new/whats-new-compose-eap.md [hunk 1]
```diff
@@ -1,22 +1,83 @@
...
+  * [UI 1.6.7](https://developer.android.com/jetpack/androidx/releases/compose-ui#1.6.1)
+  * [Foundation 1.6.7](https://developer.android.com/jetpack/androidx/releases/compose-foundation#1.6.1)
+  * [Material 1.6.7](https://developer.android.com/jetpack/androidx/releases/compose-material#1.6.1)
+  * [Material3 1.2.1](https://developer.android.com/jetpack/androidx/releases/compose-material3#1.2.0)
+* Lifecycle libraries `org.jetbrains.androidx.lifecycle:lifecycle-*:2.8.0-rc01`. Based on [Jetpack Lifecycle 2.8.0-rc01](https://developer.android.com/jetpack/androidx/releases/lifecycle#2.8.0-rc01).
+* Navigation libraries `org.jetbrains.androidx.navigation:navigation-*:2.7.0-alpha04`. Based on [Jetpack Navigation 2.7.7](https://developer.android.com/jetpack/androidx/releases/navigation#2.7.7).
+
+## Breaking changes
+
+### New Compose compiler Gradle is required for Kotlin 2.0
```
Comment (terrakok) [inline topics/whats-new/whats-new-compose-eap.md:26]: New Compose compiler Gradle **plugin**

Comment (MatkovIvan) [reply topics/whats-new/whats-new-compose-eap.md:26]:
```suggestion
### New Compose compiler Gradle plugin is required for Kotlin 2.0
```

Comment (MatkovIvan) [reply topics/whats-new/whats-new-compose-eap.md:26]:
```suggestion
### New Compose compiler Gradle plugin is required for Kotlin 2.0
```

**Review by MatkovIvan — APPROVED** [review]

File: topics/whats-new/whats-new-compose-eap.md [hunk 2]
```diff
@@ -36,17 +97,97 @@ The library was primarily introduced to support common navigation functionality,
...
+(see [the migration guide](https://www.jetbrains.com/help/kotlin-multiplatform-dev/compose-compiler.html#migrating-a-compose-multiplatform-project) for details).
+
+## Desktop
+
+### Basic support of BasicTextField2
+
+The `BasicTextField2` Compose component is now supported on a base level for desktop targets.
+Use it if your project absolutely requires it, or to test it out, but keep in mind that there may be uncovered edge cases.
+For example, `BasicTextField2` does not support IME events right now, so you won't be able to use virtual keyboards
+for Chinese, Japanese, and Korean.
```
Comment (koshachy) [inline topics/whats-new/whats-new-compose-eap.md:115]: IDK, will be more readable if we add "languages" in the end of the sentence?
Or it's better to keep it concise?

File: topics/whats-new/whats-new-compose-eap.md [hunk 3]
```diff
@@ -36,17 +97,97 @@ The library was primarily introduced to support common navigation functionality,
...
+## Desktop
+
+### Basic support of BasicTextField2
+
+The `BasicTextField2` Compose component is now supported on a base level for desktop targets.
+Use it if your project absolutely requires it, or to test it out, but keep in mind that there may be uncovered edge cases.
+For example, `BasicTextField2` does not support IME events right now, so you won't be able to use virtual keyboards
+for Chinese, Japanese, and Korean. 
+
+Full support and support for other platforms is planned for the 1.7.0 release of Compose Multiplatform.
```
Comment (koshachy) [inline topics/whats-new/whats-new-compose-eap.md:117]: Let's get read of "of".

Full support and support for other platforms is planned for(or in?) the Compose Multiplatform 1.7.0 release.

File: topics/whats-new/whats-new-compose-eap.md [hunk 4]
```diff
@@ -36,17 +97,97 @@ The library was primarily introduced to support common navigation functionality,
...
+### Basic IME keyboard support
+
+The web target for Compose Multiplatform now has basic support for virtual (IME) keyboards.
+
+## Known issues
+
+### MissingResourceException
+
+You may experience the `org.jetbrains.compose.resources.MissingResourceException: Missing resource with path: ...` error
+after changing your Kotlin version from 1.9 to 2.0 (or the other way around).
```
Comment (koshachy) [inline topics/whats-new/whats-new-compose-eap.md:162]:
```suggestion
after changing your Kotlin version from 1.9.x to 2.0.0 (or the other way around).
```

File: topics/whats-new/whats-new-compose-eap.md [hunk 5]
```diff
@@ -36,17 +97,97 @@ The library was primarily introduced to support common navigation functionality,
...
+
+In previous versions, it wasn't possible to upload a macOS app to the App Store without including an Intel version. 
+You can now set a minimum macOS version for your app among platform-specific Compose Multiplatform options:
+
+```kotlin
+compose.desktop {
+    application {
+        nativeDistributions {
+            macOS {
+              minimumSystemVersion = "12.0"
```
Comment (koshachy) [inline topics/whats-new/whats-new-compose-eap.md:177]:
```suggestion
                minimumSystemVersion = "12.0"
```

**Review by koshachy — CHANGES_REQUESTED** [review]
Some small changes here and there.
Please, take a look.

File: topics/whats-new/whats-new-compose-eap.md [hunk 6]
```diff
@@ -1,22 +1,83 @@
...
+  * [UI 1.6.7](https://developer.android.com/jetpack/androidx/releases/compose-ui#1.6.7)
+  * [Foundation 1.6.7](https://developer.android.com/jetpack/androidx/releases/compose-foundation#1.6.7)
+  * [Material 1.6.7](https://developer.android.com/jetpack/androidx/releases/compose-material#1.6.7)
+  * [Material3 1.2.1](https://developer.android.com/jetpack/androidx/releases/compose-material3#1.2.1)
+* Lifecycle libraries `org.jetbrains.androidx.lifecycle:lifecycle-*:2.8.0-rc02`. Based on [Jetpack Lifecycle 2.8.0-rc01](https://developer.android.com/jetpack/androidx/releases/lifecycle#2.8.0-rc01).
+* Navigation libraries `org.jetbrains.androidx.navigation:navigation-*:2.7.0-alpha05`. Based on [Jetpack Navigation 2.7.7](https://developer.android.com/jetpack/androidx/releases/navigation#2.7.7).
+
+## Breaking changes
+
+### New Compose compiler Gradle pliugin is required for Kotlin 2.0
```
Comment (hfhbd) [inline topics/whats-new/whats-new-compose-eap.md:26]:
```suggestion
### New Compose compiler Gradle plugin is required for Kotlin 2.0
```

File: topics/whats-new/whats-new-compose-eap.md [hunk 7]
```diff
@@ -1,22 +1,83 @@
...
+  * [UI 1.6.7](https://developer.android.com/jetpack/androidx/releases/compose-ui#1.6.7)
+  * [Foundation 1.6.7](https://developer.android.com/jetpack/androidx/releases/compose-foundation#1.6.7)
+  * [Material 1.6.7](https://developer.android.com/jetpack/androidx/releases/compose-material#1.6.7)
+  * [Material3 1.2.1](https://developer.android.com/jetpack/androidx/releases/compose-material3#1.2.1)
+* Lifecycle libraries `org.jetbrains.androidx.lifecycle:lifecycle-*:2.8.0-rc02`. Based on [Jetpack Lifecycle 2.8.0-rc01](https://developer.android.com/jetpack/androidx/releases/lifecycle#2.8.0-rc01).
+* Navigation libraries `org.jetbrains.androidx.navigation:navigation-*:2.7.0-alpha05`. Based on [Jetpack Navigation 2.7.7](https://developer.android.com/jetpack/androidx/releases/navigation#2.7.7).
+
+## Breaking changes
+
+### New Compose compiler Gradle plugin is required for Kotlin 2.0
```
Comment (koshachy) [inline topics/whats-new/whats-new-compose-eap.md:26]:
```suggestion
### New Compose compiler Gradle plugin is required for Kotlin 2.0.0
```

File: topics/whats-new/whats-new-compose-eap.md [hunk 8]
```diff
@@ -36,17 +97,107 @@ The library was primarily introduced to support common navigation functionality,
...
+        nativeDistributions {
+            macOS {
+                minimumSystemVersion = "12.0"
+            }
+        }
+    }
+}
+```
+
+See [the pull request](https://github.com/JetBrains/compose-multiplatform/pull/4271) for details.
```
Comment (koshachy) [inline topics/whats-new/whats-new-compose-eap.md:176]: Usually we don't make articles as a part of a link

**Review by koshachy — APPROVED** [review]
Minor things, overall is good
